### PR TITLE
Temporal Augmentation placeholder

### PR DESCRIPTION
**Pull request by ivanzvonkov**
Initial go at adding temporal augmentation

### CODE REVIEW

**Comment by ivanzvonkov** [comment]
Still thinking about how this should be trained though

File: src/utils/data_transform.py [hunk 1]
```diff
@@ -105,7 +112,23 @@ def transform(self, image_stack, mask=None, return_unnormalized_numpy=False):
...
+        image_stack, mask = self.normalize_and_torchify(image_stack, mask)
+
+        if self.is_train and self.temporal_augmentation and np.random.rand() < 0.8:
+            assert (
+                len(image_stack.shape) == 3
+            ), f"Expecting an image stack with shape (Temporal, Bands, Pixels) but got shape {image_stack.shape}"
+
+            synthetic_image_stack = np.zeros(image_stack.shape)
+            # Go through pixel
+            for i in image_stack.shape[2]:
```
Comment (joos2010kj) [inline src/utils/data_transform.py]: Do you mean:

for i in range(image_stack.shape[2]): 

?

**Review by joos2010kj — CHANGES_REQUESTED** [review]
Looks good

File: src/utils/data_transform.py (hunk 1 above)
Comment (ivanzvonkov) [reply src/utils/data_transform.py]: yes, thank you, fixed!

**Review by joos2010kj — APPROVED** [review]

**Pull request merged**